### PR TITLE
fix: CRW-3315

### PR DESCRIPTION
**Pull request by tolusha**
## What does this pull request change?
* clean up resources fields
* checks if checluster resources exist before installing

## What issues does this pull request fix or reference?
https://issues.redhat.com/browse/CRW-3315

## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [x] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/main/src/constants.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

**Review by azatsarynnyy — APPROVED** [review]

File: modules/administration-guide/pages/installing-che-on-openshift-using-cli.adoc [hunk 1]
```diff
@@ -15,6 +15,17 @@ You can install {prod-short} on OpenShift.
 
 * `{prod-cli}`. See: xref:installing-the-chectl-management-tool.adoc[].
 
+[IMPORTANT]
+====
+Ensure that there are no `CheCluster` API by running the following command:
+[source,subs="+quotes,attributes"]
+----
+$ {orch-cli} get crd checlusters.org.eclipse.che
+Error from server (NotFound): customresourcedefinitions.apiextensions.k8s.io "checlusters.org.eclipse.che" not found
```
Comment (max-cx) [inline modules/administration-guide/pages/installing-che-on-openshift-using-cli.adoc:24]: Does running `dsc` result in exactly the same message? In particular the `"checlusters.org.eclipse.che"` part?

File: modules/administration-guide/pages/installing-che-on-openshift-using-cli.adoc [hunk 2]
```diff
@@ -15,6 +15,17 @@ You can install {prod-short} on OpenShift.
 
 * `{prod-cli}`. See: xref:installing-the-chectl-management-tool.adoc[].
 
+[IMPORTANT]
+====
+Ensure that there are no `CheCluster` API by running the following command:
```
Comment (max-cx) [inline modules/administration-guide/pages/installing-che-on-openshift-using-cli.adoc:20]: Is this a step in the procedure?
We're asking the reader to do something `important`, but it's not in the procedure.

File: modules/administration-guide/pages/installing-che-on-openshift-using-the-web-console.adoc [hunk 3]
```diff
@@ -13,6 +13,16 @@ This section describes how to install {prod-short} using the OpenShift web conso
 
 * An OpenShift web console session by a cluster administrator. See link:https://docs.openshift.com/container-platform/{ocp4-ver}/web_console/web-console.html[Accessing the web console].
 
+[IMPORTANT]
+====
+Ensure that there are no `CheCluster` API by running the following command:
+[source,subs="+quotes,attributes"]
+----
+$ {orch-cli} get crd checlusters.org.eclipse.che
```
Comment (max-cx) [inline modules/administration-guide/pages/installing-che-on-openshift-using-the-web-console.adoc:21]: This procedure uses the OpenShift web console. Is there a GUI alternative to running this CLI command? If so, we can incorporate it into the procedure as a step (like a manual check on some UI page) in the web console.

File: modules/administration-guide/pages/installing-che-on-openshift-using-cli.adoc (hunk 1 above)
Comment (tolusha) [reply modules/administration-guide/pages/installing-che-on-openshift-using-cli.adoc:24]: `orch-cli` stands for `kubectl/oc`. The output is the same for both tools.

**Pull request merged**